WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -60,6 +60,13 @@ struct NewOrder: View {
 
                         Spacer(minLength: Layout.sectionSpacing)
 
+                        Group {
+                            OrderPaymentSection(viewModel: viewModel.paymentDataViewModel)
+
+                            Spacer(minLength: Layout.sectionSpacing)
+                        }
+                        .renderedIf(viewModel.shouldShowPaymentSection)
```

Comment:
nit: when we have to add Group to use `renderedIf` - isn't it simpler to just write `if x { }`? 🙂

Reply:
True :)